mongo/ensure-cluster-user/ensure_user.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+nodes=$(echo $MONGO_NODES | tr "," "\n")
+masterMarker='"ismaster" : true'
+
+for node in $nodes
+do
+    checkMaster="mongo $node --username ${ADMIN_USERNAME:?} --password ${ADMIN_PASSWORD:?} --authenticationDatabase ${ADMIN_DB:?} --quiet --eval 'rs.isMaster()'"
```

Comment:
I think you could have connected directly to master, without the loop
https://stackoverflow.com/a/28675561